content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/configure-enforcer-options-during-installation.md

```diff
@@ -0,0 +1,64 @@
+weight: 70
+toc: true
+---
+
+> _This document relates to Chainguard Enforce. In order to follow along, you will need access to Chainguard Enforce. You can request access through selecting **Chainguard Enforce for Kubernetes** on the [inquiry form](https://www.chainguard.dev/get-demo?utm_source=docs)._
+
+There are currently a limited list of enforcer options that can be configured when installing Chainguard into a current Kubernetes cluster. This guide will walk you through each of these Enforcer configuration settings. 
+
+Our [getting started guide](../chainguard-enforce-user-onboarding/) provides more detailed information on how to set up Chainguard Enforce, and this document provides a reference on how to configure different behaviors for your cluster. 
+
+We will use `chainctl`, the command line tool for working with Chainguard products.
+
+## Enforcer Options
+
+You'll need access to Chainguard Enforce (sign up via the [inquiry form](https://www.chainguard.dev/get-demo?utm_source=docs)), with a group and cluster already set up.
+
+Regardless of the method you choose, you'll need `chainctl` installed, which you can achieve by following our [How to Install `chainctl` tutorial](/chainguard/chainguard-enforce/how-to-install-chainctl/).
+
+The list of available Enforcer options are detailed below:
+
+* `webhook_fail_open`: is a flag to enable/disable a fail open behavior for the Enforcer webhooks. Default is set to `false`.
+* `enable_cip_cache`: is a flag to enable/disable CIP caching. Default is set to `false`.
+* `namespace_enforcement_mode`: defines the behavior of the Enforcer webhook's label selector. This option accepts two possible values: `opt-out` and `opt-in`. `opt-in` sets a behavior to the Enforcer webhooks where only labeled (`policy.sigstore.dev/include=true`) namespaces will be verified to enforce the defined policies. `opt-out` sets an opposite behavior to the Enforcer webhooks where only labeled (`policy.sigstore.dev/exclude=true`) namespaces will be **excluded** from any verification related to the defined policies. Default is set to `opt-in`.
+
+## Install with `chainctl`
+
+To install with `chainctl`, first authenticate into `chainctl` before running a command.
+
+```sh
+chainctl auth login
+```
+
+With your cluster already set up, you'll install the Chainguard Enforce Agent with `chainctl` and use the flag `--opt` to set any of your Enforcer specific settings. 
+
+For this example on a GKE, EKS, or AKS cloud infrastructure, we enable our cluster webhook to fail open by using the next command. 
+
+```sh
+chainctl cluster install --group=$GROUP_ID --context $CLUSTER --opt=webhook_fail_open=true
+```
+
+In this other example using a _private_ or on-prem cluster, we configure our Chainguard cluster to enable the cluster image policy cache and set `opt-out` as enforcement mode.
+
+```sh
+chainctl cluster install --group=$GROUP_ID --private --context $CLUSTER --opt=webhook_fail_open=true --opt=namespace_enforcement_mode=opt-out
```

Comment:
I am a newbie here, should we add a paragraph to explain how to verify whether the changes had the desired effect ?